components/ui/stickerItems.tsx

```diff
@@ -0,0 +1,33 @@
+export const stickerItems = [
+  { id: 1, path: "/img/sticker/notSelect.png", bgPath: "/img/bg/4-16.png" },
+  {
+    id: 2,
+    path: "/img/sticker/sticker1.png",
+    bgPath: "/img/bg/4-17/bg-sticker1.png",
+  },
+  {
+    id: 3,
+    path: "/img/sticker/sticker2.png",
+    bgPath: "/img/bg/4-17/bg-sticker2.png",
+  },
+  {
+    id: 4,
+    path: "/img/sticker/sticker3.png",
+    bgPath: "/img/bg/4-17/bg-sticker3.png",
+  },
+  {
+    id: 5,
+    path: "/img/sticker/sticker4.png",
+    bgPath: "/img/bg/4-17/bg-sticker4.png",
+  },
+  {
+    id: 6,
+    path: "/img/sticker/sticker5.png",
+    bgPath: "/img/bg/4-17/bg-sticker5.png",
+  },
+  {
+    id: 7,
+    path: "/img/sticker/sticker6.png",
+    bgPath: "/img/bg/4-17/bg-sticker6.png",
+  },
+];
```

Comment:
1. This is not `ui` but rather a `constant`. Please put this in `constant` folder.
2. add `as const` to the end of the object to enable typescript to infer types staticly.
```ts
const a = [
...
] as const;
```